Apps/Sandcastle/gallery/development/ClassificationPrimitive.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Draw the intersection of a volume and a photgrammetry dataset.">
+    <meta name="cesium-sandcastle-labels" content="Development">
```

Comment:
Are you sure this should go under `Development`?  I would rather it be more prominent.